proto/viam/service/motion/v1/motion.proto

```diff
@@ -94,10 +109,8 @@ message MoveOnGlobeRequest {
   common.v1.ResourceName movement_sensor_name = 5;
   // Obstacles to be considered for motion planning
   repeated common.v1.GeoObstacle obstacles = 6;
-  // Optional linear velocity to target when moving
-  optional float linear_meters_per_sec = 7;
-  // Optional angular velocity to target when turning
-  optional float angular_deg_per_sec = 8;
```

Comment:
These field names and indexes should be reserved, like here: https://github.com/viamrobotics/api/pull/312/files